1_assignment/second_largest.js

```diff
@@ -0,0 +1,31 @@
+
+array= [ -1 , -4 , 2 , -10]
+
+
+function secondLargest(array) {
+  // Write your code here
+  var i;
+  var biggest=null;
+  var secondBiggest=null;
+  for (i=0;i<array.length;i++)
+    {
```

Comment:
Opening braces should start from the same line.
Formatting required: Spaces around '=' after ';' and around '<'